### PR TITLE
Feature/tas 23

### PR DESCRIPTION
**Pull request by marczahnlls**
@mickadoo

### CODE REVIEW

File: src/LooplineSystems/CloseIoApiWrapper/Api/CustomFieldApi.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    {
+        /** @var CustomField[] $customFields */
+        $customFields = array();
+
+        $apiRequest = $this->prepareRequest('get-customFields');
+
+        /** @var CloseIoResponse $result */
+        $result = $this->triggerGet($apiRequest);
+
+        if ($result->getReturnCode() == 200) {
```
Comment (mickadoo) [inline src/LooplineSystems/CloseIoApiWrapper/Api/CustomFieldApi.php]: what happens if it fails?

File: src/LooplineSystems/CloseIoApiWrapper/Api/LeadApi.php [hunk 2]
```diff
@@ -58,6 +58,46 @@ public function getAllLeads()
...
+
+        return $leads;
+    }
+
+    /**
+     * @param array $params
+     *
+     * @return string
+     */
+    private function buildQueryString(array $params)
```
Comment (mickadoo) [inline src/LooplineSystems/CloseIoApiWrapper/Api/LeadApi.php]: i suppose this could go in the abstract api class

**Comment by mickadoo** [comment]
@marczahnlls do you know why the build is failing on travis?

File: src/LooplineSystems/CloseIoApiWrapper/Api/LeadApi.php (hunk 2 above)
Comment (marczahnlls) [inline src/LooplineSystems/CloseIoApiWrapper/Api/LeadApi.php]: Yes - But let's do this maybe later. For now I want to have the crm proxy gone live.

File: src/LooplineSystems/CloseIoApiWrapper/Api/CustomFieldApi.php (hunk 1 above)
Comment (marczahnlls) [inline src/LooplineSystems/CloseIoApiWrapper/Api/CustomFieldApi.php]: https://github.com/loopline-systems/closeio-api-wrapper/blob/develop/src/LooplineSystems/CloseIoApiWrapper/Library/Curl/Curl.php#L65-L69

**Comment by mickadoo** [comment]
Would be nice to have the tests passing again, but other than that code review done

**Comment by marczahnlls** [comment]
The tests themselfs are running.

**Pull request merged**